### ISSUE TITLE
Some Mod Keys stop working overtime

### ISSUE THREAD

**Issue by DanitiKain** [LukeSmithxyz/LARBS#50]
So far I have noticed mod+x(lock screen) and mod+u(drop down terminal) stop working over time.  Not sure what is causing this as other mod+keys work fine.   Like ranger and terminal and things I use on a normal basis still work.

**Comment by DanitiKain** [comment]
ignore mod+x I figured out the problem was somehow the config was edited.  However mod+u issue still exists.

**Comment by LukeSmithxyz** [comment]
Does the dropdown window ever work? Are you exiting out with `ctrl-d` or something? (You shouldn't do that).

Does `mod+a` bring up the dropdown calculator? It would help knowing if it were a more general problem.

**Comment by Serverislife** [comment]
I am not seeing the mod+x issue or the mod+u issue. However, I would like to note that mod+F1 only works for about 30 seconds after boot. Let me know if I can provide any information to help you triage this.

**Comment by DanitiKain** [comment]
Yes mod+a still works and I have never hit ctrl d as I didn't even know that it was an option.

**Comment by Serverislife** [comment]
Hey all,

Obviously this isn't a detrimental issue, so I'm not going to push hard for a fix, but I was just wondering, Luke, is there anything I can do to help out with this issue?
